airflow_clickhouse_plugin/operators/clickhouse_operator.py

```diff
@@ -1,18 +1,21 @@
 from typing import *
 
 from airflow.models import BaseOperator
+from airflow.utils.decorators import apply_defaults
 
 from airflow_clickhouse_plugin.hooks.clickhouse_hook import ClickHouseHook
 
 
 class ClickHouseOperator(BaseOperator):
```

Comment:
Is there `DbApiOperator` present in Airflow?

Reply:
No, the DbApiOperator does not exist.  We have to use the BaseOperator as superclass here.